diff --git a/CMakeLists.txt b/CMakeLists.txt
index cc82669f..4bb87146 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -595,15 +595,6 @@ else()
 	include_directories(${PKGDEPS_INCLUDE_DIRS})
 endif()
 
-#===============================================================================
-#= Release flags
-#-------------------------------------------------------------------------------
-set(CMAKE_CXX_FLAGS "-Wall -Wextra")
-set(CMAKE_BUILD_TYPE Release)
-set(CMAKE_CXX_FLAGS_DEBUG "-g")
-set(CMAKE_CXX_FLAGS_RELEASE "-O3")
-add_compile_options(-O3)
-
 #===============================================================================
 #= Linker Configuration
 #-------------------------------------------------------------------------------
